package.json

```diff
@@ -15,7 +16,8 @@
   "dependencies": {
     "@replit/database": "^3.0.1",
```

Comment:
TODO: Move to Dev-Dependencies because it will only be needed for the migration script. Can be completly removed in a future PR.